index.js

```diff
@@ -45,7 +45,7 @@ exports.handler = function(event, _context, callback) {
                     return new Promise(() => {})  // the next then-blocks will never be executed
             }
 
-            return img.withoutEnlargement().toBuffer();
+            return img.withoutEnlargement().rotate().toBuffer();
```

Comment:
[v0.21 Changelog](http://sharp.dimens.io/en/latest/changelog/#v0210-4th-october-2018) says that `.withoutEnlargement`, `.max`, and `.min` are deprecated.
It seems we need to change `.resize` method above on [:L30](https://github.com/sagidM/s3-resizer/blob/master/index.js#L30) to
`.resize(..., ..., {withoutEnlargement: true, fit})`.
Where `fit` is `null | 'inside' | 'outside'`. I.e.
```js
var fit;
switch (func) {
  case 'max': fit = 'inside'; break;
  case 'min': fit = 'outside'; break;
  case null: fit = null; break;
  default: // ...
}
```

And could you please explain why should we have calling `.rotate()` here?

Reply:
Sure thing - will do those changes. rotate function should rotate the image according to EXIF info: https://sharp.pixelplumbing.com/en/stable/api-operation/#rotate. the image I've attached is from an Android device - when you view the original image it's in portrait. but when I've resized it with your library it's resized to a landscape image. to fix this I've added the portrait call - that will rotate the image back to original position using EXIF info.